README.md

```diff
@@ -17,29 +17,35 @@ You can install the provider using `pip`:
 pip install qiskit-alice-bob-provider
 ```
 
-`pip` will handle installing all the python dependencies automatically and you
-will always install the  latest (and well-tested) version.
+`pip` will handle installing all the python dependencies automatically and 
+will always install the latest (and well-tested) version.
 
-## Remote execution on Alice & Bob QPUs: use your API key
+The provider is currently **NOT** compatible with Qiskit 1.0. We recommend
+creating a new environment before installing it.
 
-To obtain an API key, please [contact Alice & Bob](https://alice-bob.com/contact/).
+## Remote execution on Alice & Bob QPUs with a Felis Cloud subscription
 
-You can initialize the Alice & Bob remote provider using your API key
-locally with:
+When subscribing to Felis Cloud on the
```

Comment:
"Subscribing to Felis Cloud allows you to generate API Keys" (not just to get one)

Reply:
I would not fix this: most users only need one key, and the text doesn't suggest you can't generate more keys.

Reply:
Fine for me then